src/coreclr/tools/aot/Mono.Linker.Tests/Mono.Linker.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);SYSLIB1045</NoWarn>
```

Comment:
Why is it not picking up test rules suppressions?

Reply:
Switching to the tests config is currently configured under libraries:
https://github.com/dotnet/runtime/blob/d92e70f05b8de72758f3a990844e53c76b44d713/src/libraries/Directory.Build.props#L47-L50